src/repositories/tweet.repository.ts

```diff
@@ -0,0 +1,37 @@
+import { ObjectId } from "mongodb";
+
+import { TweetModel } from "../models/Tweet";
+
+export const getOneById = (id: ObjectId) => {
+  return TweetModel.findById(id);
+};
```

Comment:
In that place we need to string-type parameters of function. And inside function take ObjectId by string. Only repository has to know how works with database

Reply:
@VitaliiDenysyuk , well, i thing that validation changing type and throwing error in case of bad input should be more on upper level like service. Besides that, we can get id not only as string, but as object id | buffer or 10 more variations in ObjectId constructor as well and overloading method for each one would be pointless

Reply:
so i dont thing that further separating services and db would be better than creating more possibilities to use repository methods

Reply:
ObjectId only for mongoDb, if we change database to sql we need to rewrite services and repositories it's not good